src/EurekaClient.js

```diff
@@ -1,9 +1,10 @@
 import request from 'request';
 import fs from 'fs';
 import yaml from 'js-yaml';
-import merge from 'deepmerge';
+import merge from 'lodash/merge';
```

Comment:
The reason I switched from `deepmerge` is because of https://github.com/KyleAMathews/deepmerge/issues/28. It caused a lot of weird issues with tests being dependant on each other (and different Eureka client would've been too) since it only clones the top level of the objects.

Lodash is much more battle tested. Besides, this is run through babel, and only the `merge` functionality is actually imported.